internal/v2/common/jx.go

```diff
@@ -0,0 +1,13 @@
+package common
+
+import "github.com/go-faster/jx"
+
+// StringToJx converts a string to a jx.Raw JSON representation.
+func StringToJx(s string) jx.Raw {
```

Comment:
ogen uses jx instead of interface{}.
These are some helper functions for when we know the json is a string.